istio-control/istio-config/templates/deployment.yaml

```diff
@@ -36,7 +36,9 @@ spec:
           - containerPort: 15019
           command:
           - /usr/local/bin/galley
+{{- if and (.Values.global.istioVersion) (semverCompare "<1.2" .Values.global.istioVersion) }}
```

Comment:
Is this because they removed 'server' in master ? I think a better fix is to ask them to revert that change..
For now I'll merge this.